src/clj/rems/db/organizations.clj

```diff
@@ -35,11 +36,16 @@
 (defn get-organizations-raw []
   (->> (db/get-organizations)
        (map parse-organization)
+       (map #(update % :organization/owners (partial mapv (comp users/get-user :userid))))
+       (map #(update % :organization/modifier (comp users/get-user :userid)))
```

Comment:
shouldn't this be done in the service layer according to our architecture? also, is this `-raw` any more?

Reply:
I had a look at the others and they seemed to use the `get-user` from `db.users` in the db-layer whereas for e.g. organization we have `join-organization` in the service layer. Here the `UserWithAttributes` schema coercing requires us to load them already at this level.

You are right about the `-raw` that perhaps it's not correct anymore. We can have a think about how we want to continue with this.

Reply:
Let's do this one in another PR once we have the idea. https://github.com/CSCfi/rems/issues/2342